src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -86,15 +88,18 @@ public FunctionDescriptor GetHttpFunctionOrNull(Uri uri)
         protected override void OnInitializeConfig(JobHostConfiguration config)
         {
             base.OnInitializeConfig(config);
-
+            _metricsLogger = new WebHostMetricsLogger();
             // Add our WebHost specific services
-            config.AddService<IMetricsLogger>(new WebHostMetricsLogger());
+            config.AddService<IMetricsLogger>(_metricsLogger);
         }
 
         protected override void OnHostStarted()
         {
             base.OnHostStarted();
-
+            if (_metricsLogger != null)
+            {
+                _metricsLogger.OnHostStartedEvent(Instance);
```

Comment:
If we want to raise this only once a day then it would require us to save last updated time stamp in site content directory which would complicate the overall design. This would be mostly fired when the worker process starts up. As raising ETW event is pretty fast and lightweight so it shouldn't cause any run time perf issues.